### DISCUSSION TITLE
Database Schema Planning

### DISCUSSION THREAD

**Discussion by SahanJDesigns** [SahanJDesigns/Project-LMS#2] [General]
## After reviewing the work completed yesterday, I suggest the following collections for our project. 

- Quiz: Stores details such as marks, timestamps, Question IDs, and other relevant quiz-related information.
- Questions: Contains the content of each question used in quizzes and assessments.
- User: Holds general user details such as user ID, name, address, and other attributes common to all roles in the LMS.
- Student: Contains specific information required to track and monitor the activities and progress of students.
- Teachers: Stores specific details necessary for managing and tracking teacher-related activities within the LMS.
- Courses: Includes information about courses, such as course names, descriptions, and associated lesson IDs.
- Lessons: Contains resources related to lessons, including video links, study materials, and additional content.
- Schedule: Maintains schedules for individual students to organize and track their learning plans.

Kindly provide your comments, along with any improvements and suggestions.🤗

**Comment by NaveenSandaruwan** [comment]
Good design 😎
